### ISSUE TITLE
Bind multiple fields to a single model method

### ISSUE THREAD

**Issue by chr-hertel** [sensiolabs-de/rich-model-forms-bundle#50]
Given a model that has a setter-method with two or more arguments

```php
class Order
{
    public function ship(Address $address, string $trackingNumber): void
    {
        $this->shippingAddress = $address;
        $this->trackingNumber = $trackingNumber;
    }
}
```

what is the best solution to this problem without changing the model to use a value object or something? is there already a good solution or do we need to come up with an idea?

**Comment by chr-hertel** [comment]
possible solution could look like this:

```php
class OrderType extends AbstractType
{
    public function buildForm(FormBuilderInterface $builder, array $options)
    {
        $builder
            // ...
            ->add('shippingAddress', AddressType::class, [
                'write_property_path' => 'ship',
                'argument_name' => 'address'
            ])
            ->add('trackingNumber', TextType::class, [
                'write_property_path' => 'ship',
            ]);
```

grouped by `write_property_path` we could collect all relevant data and pass to method by field name. optionally one could override the field name using the `argument_name` option

**Comment by chr-hertel** [comment]
`argument_name` could be reused for constructor arguments by the `factory` options as well

**Comment by xabbuh** [comment]
WIP PR is ready to be reviewed at #56

**Comment by xabbuh** [comment]
implemented in #56
